src/bo4e/bo/messlokation.py

```diff
@@ -83,3 +85,7 @@ class Messlokation(Geschaeftsobjekt):
     Alternativ zu einer postalischen Adresse und Geokoordinaten kann hier eine Ortsangabe mittels Gemarkung und
     Flurstück erfolgen.
     """
+    #: Lokationszuordnung, um bspw. die zugehörigen Messlokationen anzugeben
```

Comment:
Nur weil wir hier schon in den Messlokationen sind ;) 
```suggestion
    #: Lokationszuordnung, um bspw. die zugehörigen Marktlokationen anzugeben
```